docs/tools/sg3utils.md

```diff
@@ -250,21 +252,21 @@ for hex use a leading '0x' or a trailing 'h'. Either the --zone=ID
 or --all option needs to be given.
 ```
 
-Resetting all sequential write zones of the disk can be done using the `--all`
-option.
+Reset all sequential write zones on the disk by using the `--all` option:
 
 ```plaintext
 # sg_reset_wp --all /dev/sdd
 ```
 
-A single sequential zone write pointer can be reset using the `--zone` option.
+Reset a single sequential zone write pointer can be by using the `--zone`
```

Comment:
can be -> can be done ? Looks like a verb is missing...

Reply:
Good catch, Damien.

I rewrote all of these directives in parallel, in the imperative mood.

So when I corrected this, I went in a different direction.

I wrote "Reset a single sequential zone write pointer by using the `--zone`"...

Again, good catch.